tests/outofrange_queries.test/queries

```diff
@@ -79,6 +80,7 @@ alter table t {schema {
     cstring c[4] null=yes
```

Comment:
Double check the intent here.  The keys referenced this column without it being present (same as above).